src/src/analytics/query.sql

```diff
@@ -0,0 +1,47 @@
+# SQL to create tables on top of parquet data stored in S3 
+# and also examples of how to query them.
+
+drop table hf_datasets
+
+CREATE EXTERNAL TABLE IF NOT EXISTS hf_datasets(
+    request_time string,
+	dataset string,
+    author string,
+    created_at timestamp,   
+    last_modified timestamp,
+    private boolean,
+    disabled boolean,
+    gated string,
+    downloads int,
+    downloads_all_time int,
+    tags ARRAY <string>, 
+    license string,
+    trending_score double
+)
+partitioned by (`date` date)
+STORED AS PARQUET
+LOCATION "s3://<bucket>/huggingface/datasets/";
```

Comment:
Is there a way to make `<bucket>` programmatically set? At the very least, is the need to define it correctly documented somewhere.